src/main/java/org/opentripplanner/routing/impl/PdxFareServiceFactory.java

```diff
@@ -1,46 +0,0 @@
-package org.opentripplanner.routing.impl;
```

Comment:
Did you happen to delete the Pdx* files rather than renaming them to HighestFare? I think it might have been easier to trace the history of changes by simply renaming them. Perhaps you did that and this is just git being difficult!

Reply:
I did, but the changes were probably too much for git to recognize.